### PR TITLE
Fix part of #1287: Fix lint issues in app module in the kotlin files (topic shared test)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of #1287
`topic shared test` folder `ktlint` check

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt [hunk 1]
```diff
@@ -79,9 +79,14 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicLessonsFragmentTest {
 
-  @Inject lateinit var context: Context
-  @Inject lateinit var profileTestHelper: ProfileTestHelper
-  @Inject lateinit var storyProgressTestHelper: StoryProgressTestHelper
+  @Inject
+  lateinit var context: Context
```
Comment (veena14cs) [inline app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt:83]: Bring this to single line.

**Review by veena14cs — APPROVED** [review]
LGTM. Suggested one nit change.

File: app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt:83]: Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting

File: app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt [hunk 2]
```diff
@@ -156,7 +161,10 @@ class TopicLessonsFragmentTest {
 
   @Test
   fun testLessonsPlayFragment_loadRatiosTopic_completeStoryProgress_isDisplayed() {
-    storyProgressTestHelper.markFullStoryPartialTopicProgressForRatios(profileId, /* timestampOlderThanAWeek= */ false)
+    storyProgressTestHelper.markFullStoryPartialTopicProgressForRatios(
+      profileId, /* timestampOlderThanAWeek= */
+      false
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt]: Nit change here and everywhere else in this file:
```
      profileId,
      /* timestampOlderThanAWeek= */ false
```

**Review by rt4914 — CHANGES_REQUESTED** [review]
One nit change suggested.

File: app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt (hunk 2 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt]: Done

**Review by veena14cs — APPROVED** [review]
You can merge this PR.

**Comment by anandwana001** [comment]
> You can merge this PR.

Thanks :)

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**